export/config.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
For the future: follow existing style. No spaces around function args; use 2-space indentation in function body.

Reply:
Alright @seanlip . Was confused because of an old copyright comment. Maybe we should change them to the new style guidelines as well?

Reply:
Sure. Will take care of this from the next time. @seanlip